### ISSUE TITLE
Little help to get in starting

### ISSUE THREAD

**Issue by FutureCow** [Stan-Gobien/PVdiary2-Docker#2]
Thanks for making this Docker!
I have some problems getting it starting, hopefully you can help me.
I changed the directories in the docker compose file, download the entrypoint.sh inside the scripts folder and chmod+x the file.
After starting the container(with docker compose) I get this error:
pvdiary-pvdiary2-1  | Entrypoint.sh start
pvdiary-pvdiary2-1  | Update script files from Github
pvdiary-pvdiary2-1  | Making sure executables are in the right locations
pvdiary-pvdiary2-1  | cp: cannot stat '/home/pvdiary2/bin/pvdiary': No such file or directory

inside de data directory i see only the folder /pvdiary2/var/pvdiary/csv is created. I don't see any other file. So somthing goed wrong getting the other scripts?

**Comment by Stan-Gobien** [comment]
I changed the entrypoint.sh script to fix that.
But now I see another issue with zip/unzip in php.

I'll keep you posted.

**Comment by Stan-Gobien** [comment]
I fixed the errors.
Should be all good now.

**Comment by Stan-Gobien** [comment]
You should delete your stack/container and start over with the instructions.
So you use the latest versions of the script files.
The built-in update mechanism needs 2 runs before the updated scripts take effect.

**Comment by FutureCow** [comment]
Thanks for the update, everything gets up running now.
Only got a problem with the autorun

> 04:00:01 TOOLBIN_CRON starting for pvdiary_autorun logfile /home/pvdiary2/logs/pvdiary_autorun.log_20230311
> 04:00:01 Config loaded - start: 2023-03-11 04:00:01   exit-at: 2023-03-11 19:14:00
> 04:00:01 Executing exec_at_start pvdiary --httpd --dashboard --start 
> 04:00:01 Job finish - log follows: sh: 1: pvdiary: not found
> 
> 04:00:01 Executing exec_at_start toolbin --cliserver --start 
> 04:00:01 Job finish - log follows: sh: 1: toolbin: not found
> 
> 04:00:01 First_run finished
> 07:01:00 Execute Import  command pvdiary --import --export --sync 
> 07:01:00 Job finish - log follows:
> sh: 1: pvdiary: not found

If I get into the container, and run pvdiary in the console, everyting is working. I can import data en generate the graphs. So only autorun is not working correct. Can you give a last helping hand for this problem?

**Comment by Stan-Gobien** [comment]
Seems an issue with the path.
Strange, I did not experience this.

Can you try by editing the autorun part in the pvdiary.cfg config file by adding the absolute path (/home/pvdiary2/bin/)
Like my example below:

```
 exit_at = "@sunset+40minutes"
 exit_at = "tomorrow 00:30"
 exec_at_start[] = "/home/pvdiary2/bin/pvdiary --httpd --dashboard --start"
 exec_at_start[] = "/home/pvdiary2/bin/pvdiary --cliserver --start"
 exec[] = "Import;@sunrise,@sunset+20minutes,15 minutes;/home/pvdiary2/bin/pvdiary --import --export --sync"
 exec[] = "Cleanup;@sunrise-20minutes, @sunrise, 1 day ;/home/pvdiary2/bin/pvdiary --cleanup"
 exec[] = "Backup;@sunset+30minutes  , 23:59,  7 days ;/home/pvdiary2/bin/toolbin pvdiary --db --backup"
 exec[] = "Rapport;sunday @sunset , 23:59, 2 days ;/home/pvdiary2/bin/pvdiary --summary --mail
```

`

**Comment by Stan-Gobien** [comment]
Actually now that I think about it, I myself used the absolute path only in some of the commands in autorun.
For the summary I did not use the absolute path and the summary mail does not work for me.
So maybe thats why.

For the exec_at_start I did not have the absolute path set, but that I didn't notice because I start that with the entrypoint.sh script every time the container starts.

I changed it all to absolute path now, the example above is strait out of my new pvdiary.cfg.
